hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/benefitInformation.jsp

```diff
@@ -24,27 +24,33 @@
 
 <div id="${n}dl-benefit-summary" class="fl-widget portlet dl-benefit-summary hrs">
   <div class="dl-banner-links">
-    <c:choose>
-    <c:when test="${enrollmentFlag == 'O'}">
+    <sec:authorize ifAnyGranted="ROLE_VIEW_BENEFITS">
       <div class="dl-banner-link">
-        <spring:message code="benefit.summary.enrollment.O.message" text="You have a benefit enrollment opportunity. Please enroll online by clicking the following link: "/>
+        You have a benefit enrollment opportunity.
         <a target="_blank" href="${hrsUrls['Open Enrollment/Hire Event']}"><spring:message code="open.enrollment" text="Open Enrollment" /></a>
         <c:if test="${isMadisonUser}">
-          ${learnMoreEBenefitGuide}
+            <a target="_blank" href="https://hr.wisc.edu/benefits/new-employee-benefits-enrollment/">Learn more</a>
         </c:if>
+        <c:if test!="${isMadisonUser}">
```

Comment:
This syntax won't work. You could maybe do `<c:if test="${not isMadisonUser}">` .  I always have to look up the JSTL syntax, I have Bayern's book in my office.

Consider using `<c:choose>` instead though. This is a classic choose-when-otherwise where you want one or the other of two items to fire. The expressive way to say that is choose-when-otherwise :)